ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -685,7 +685,19 @@ function upgrade() {
         clear_upgrade
         check_upgrade_status "${cluster_src_ver}"
     fi
-    run_command "oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade --force=${FORCE_UPDATE}"
+
+    cmd="oc adm upgrade --to-image=${TARGET} --allow-explicit-upgrade"
+    if [[ "$TARGET_VERSION" != *"nightly"* ]]; then
+        conditionalUpdates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]?.version')
```

Comment:
Do you have an example output of `oc get clusterversion version -o json | jq -r '.status.conditionalUpdates` ?

Reply:
also cc @jiajliu to help confirm.

Reply:
sorry, you are right